### DISCUSSION TITLE
animated markers

### DISCUSSION THREAD

**Discussion by rawDRAW** [keiwando/toonsquid#528] [Ideas]
hi, I would like to be able to use animated markers so that I can store multiple animations in one symbol

https://github.com/keiwando/toonsquid/assets/159346218/78e0611d-ae98-45fd-958e-ea37ff6be59c

**Comment by rawDRAW** [comment]
this is for example, we have a ship that lands, after which its engines should turn off

https://github.com/keiwando/toonsquid/assets/159346218/131b814d-db0b-4f05-8fae-091ac6fdbe53

**Comment by rawDRAW** [comment]
Now if you try to use a marker in a looping animation, it won't work correctly. thank you for the opportunity to write sentences, and sorry for the roughness of my speech, I don't speak English very well

https://github.com/keiwando/toonsquid/assets/159346218/5b9f3f9c-948a-402a-8c8f-b3af152543c5

**Comment by invishane** [comment]
I have a similar issue. I have a cat body standing and then walking. When you use markers, they are set to HOLD type keyframe. A work around that i have been using is to create an animation symbol INSIDE the main timeline symbol. Then, you can copy paste where you want to make changes on the SAME track. Watch this video I hope it helps. It works. What you are suggesting/requesting would be MUCH MUCH easier, but this is only thing i have found to work so far!  Link to VIDEO:

https://www.dropbox.com/scl/fi/cgwh6cj0cwj87hhc1hcxs/Video-Feb-25-2024-3-58-54-PM.mov?rlkey=x2xciczs58dmb3mhu9w1hik6j&dl=0
